packages/wdio-types/src/Options.ts

```diff
@@ -349,6 +349,22 @@ export interface Testrunner extends Hooks, Omit<WebdriverIO, 'capabilities'>, We
      * ```
      */
     capabilities: RemoteCapabilities
+    /**
+     * Defines a set of capabilities to be applied to all platforms.
+     * ```
+     * // wdio.conf.js
+     * export const config = {
+     *   // multiremote example
+     *   commonCapabilities: {
+     *     'bstack:options': {
+     *        projectName: "Your static project name goes here",
+     *        buildName: "Your static build/job name goes here"
+     *      }
+     *    }
+     * })
+     * ```
+     */
+    commonCapabilities?: WebdriverIO.Capabilities
```

Comment:
why do we need this change?

Reply:
Surprisingly, `config: Options.Testrunner` object from webdriverio, is sending capabilities under `commonCapabilities`. We have to send user's config to CLI. So, to read user caps, needed to add it in type.